src/internal/ajax/ajax.ts

```diff
@@ -335,99 +328,95 @@ export function fromAjax<T>(config: AjaxConfig): Observable<AjaxResponse<T>> {
 
     let xhr: XMLHttpRequest;
 
-    try {
```

Comment:
Here the big `try/catch` that was around everything was removed. That caused the code to shift, but was the only change.